<launch>
  <node name="node_manager" pkg="cafer_server" type="node_manager" ns="cafer" />
  <node name="getid" pkg="cafer_server" type="getid" ns="cafer" />
  <node name="launch_node" pkg="cafer_server" type="launch_node" ns="cafer"/>
</launch>
